src/commands/scmt/test/run.ts

```diff
@@ -0,0 +1,187 @@
+import { flags } from '@oclif/command'
+import { core } from '@salesforce/command'
+import CommandBase from '../../../command_base'
+
+// Initialize Messages with the current plugin directory
+core.Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = core.Messages.loadMessages('@desklabs/scmt', 'test');
+// const FIELDS = ['AccountRecordsFailed__c', 'AccountRecordsMigrated__c', 'AccountRecordsTotal__c', 'ArticleRecordsFailed__c', 'ArticleRecordsMigrated__c', 'ArticleRecordsTotal__c', 'AttachmentRecordsFailed__c', 'AttachmentRecordsMigrated__c', 'AttachmentRecordsTotal__c', 'CaseRecordsFailed__c', 'CaseRecordsMigrated__c', 'CaseRecordsTotal__c', 'ContactRecordsFailed__c', 'ContactRecordsMigrated__c', 'ContactRecordsTotal__c', 'InteractionRecordsFailed__c', 'InteractionRecordsMigrated__c', 'InteractionRecordsTotal__c', 'NoteRecordsFailed__c', 'NoteRecordsMigrated__c', 'NoteRecordsTotal__c', 'TopicRecordsFailed__c', 'TopicRecordsMigrated__c', 'TopicRecordsTotal__c', 'RecordsFailed__c', 'RecordsMigrated__c', 'RecordsTotal__c', 'Config__c', 'Desk_Config_Id__c', 'Desk_Endpoint__c', 'Error__c', 'JobId__c', 'Log__c', 'Mapping__c', 'Migration_Type__c', 'Object__c', 'StartDate__c', 'Status__c'];
+
+export default class Run extends CommandBase {
```

Comment:
I'm showing a problem in vscode with this line.  

`{
	"resource": "/Users/khaver/code/salesforce/sfdx-plugin-scmt/src/commands/scmt/test/run.ts",
	"owner": "typescript",
	"code": "2417",
	"severity": 8,
	"message": "Class static side 'typeof Run' incorrectly extends base class static side 'typeof CommandBase'.\n  Types of property 'flagsConfig' are incompatible.\n    Type '{ endpoint: IOptionFlag<string>; username: IOptionFlag<string>; password: IOptionFlag<string>; mi...' is not assignable to type 'SfdxFlagsConfig'.\n      Property 'endpoint' is incompatible with index signature.\n        Type 'IOptionFlag<string>' is not assignable to type 'string | boolean | IBooleanFlag<any> | IOptionFlag<any> | JsonMap'.\n          Type 'IOptionFlag<string>' is not assignable to type 'JsonMap'.\n            Index signature is missing in type 'IOptionFlag<string>'.",
	"source": "ts",
	"startLineNumber": 13,
	"startColumn": 22,
	"endLineNumber": 13,
	"endColumn": 25
}`

Reply:
@kylehaver does it give a problem compiling or running?

Reply:
I have the same error, but it runs.